frontend/app/src/components/helper/PrioritizedList.vue

```diff
@@ -165,78 +164,63 @@ const autoCompleteHint: ComputedRef<string> = computed(() => {
         <tbody>
           <tr v-if="noResults">
             <td colspan="4">
-              <VRow class="pa-3 text-h6" justify="center">
-                <VCol cols="auto">
-                  {{ t('prioritized_list.item.empty', itemNameTr) }}
-                </VCol>
-              </VRow>
+              <div class="flex justify-center pa-3 text-h6">
+                {{ t('prioritized_list.item.empty', itemNameTr) }}
+              </div>
             </td>
           </tr>
           <tr v-for="(identifier, index) in value" :key="identifier">
             <td>
               <div class="flex flex-col py-2">
-                <div>
-                  <VBtn
-                    :id="'move-up-' + identifier"
-                    icon
-                    small
-                    :disabled="isFirst(identifier)"
-                    @click="move(identifier, false)"
-                  >
-                    <VIcon>mdi-chevron-up</VIcon>
-                  </VBtn>
-                </div>
-                <div>
-                  <VBtn
-                    :id="'move-down-' + identifier"
-                    icon
-                    small
-                    :disabled="isLast(identifier)"
-                    @click="move(identifier, true)"
-                  >
-                    <VIcon>mdi-chevron-down</VIcon>
-                  </VBtn>
-                </div>
+                <RuiButtonGroup variant="outlined" size="sm" icon vertical>
+                  <template #default>
```

Comment:
just to be sure, do you need to put it in default slot this way?

Reply:
There is a bug with <ButtonGroup> and <RadioGroup>, the children aren't reactive.
We need to do this until we move to vue 3